packages/ui/src/assets/svg/reown-logo.ts

```diff
@@ -0,0 +1,14 @@
+import { svg } from 'lit'
+
+export const reownSvg = svg`<svg width="60" height="16" viewBox="0 0 60 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Maybe not for this PR but we could remove all `xmlns` in the code as we don't need it really